src/main/scala/org/broadinstitute/dsde/workbench/sam/audit/SamAuditModel.scala

```diff
@@ -51,10 +51,10 @@ object SamAuditModelJsonSupport {
     }
   }
 
-  implicit val AuditInfoFormat = jsonFormat2(AuditInfo)
+  implicit val AuditInfoFormat: RootJsonFormat[AuditInfo] = jsonFormat2(AuditInfo)
```

Comment:
as of Scala 2.13.11, this caused compile to break. It's a warning, but we have `-Wconf … any:e` set, which causes warnings to be errors.

The error was "Implicit definition should have explicit type" and we had many instances of this issue; lots of lines of code changed below.